dockerfiles/alpine/is/init.sh

```diff
@@ -16,53 +16,15 @@
 # ------------------------------------------------------------------------
 set -e
 
-# product profile variable
-wso2_server_profile=is
-
-# custom WSO2 non-root user and group variables
-user=wso2carbon
-group=wso2
-
-# file path variables
+# volume mounts
+config_volume=${WORKING_DIRECTORY}/wso2-config-volume
 artifact_volume=${WORKING_DIRECTORY}/wso2-artifact-volume
-config_map_volume=${WORKING_DIRECTORY}/wso2-config-volume
-temp_shared_artifacts=${WORKING_DIRECTORY}/wso2-tmp/deployment
-original_shared_artifacts=${WSO2_SERVER_HOME}/repository/deployment
-
-# a grace period for mounts to be setup
```

Comment:
@chamindi-a the implementation between lines 33 and 63 is required for attaching pod volumes in Kubernetes. This should not be removed and should be re-instated.